static/src/stylesheets/module/commercial/_creatives.scss

```diff
@@ -1,5 +1,7 @@
 .creative__tracking-pixel {
     position: absolute;
+    width: 1px;
```

Comment:
I think with this argument you just shot yourself in the foot. Let's pursue this discussion over a beer. What say we and @uplne and whoever else wants to join go for a drink this week?